src/mlpack/methods/lsh/lsh_search_impl.hpp

```diff
@@ -8,6 +8,12 @@
 #define MLPACK_METHODS_NEIGHBOR_SEARCH_LSH_SEARCH_IMPL_HPP
 
 #include <mlpack/core.hpp>
+#include <vector> //TODO: included?
+#include <queue> //TODO: included?
```

Comment:
Yep, these should already be included by prereqs.hpp, so you should be able to remove these. (If not, we should add them to prereqs.hpp).